test/integration/multitenant/multitenant_test.go

```diff
@@ -118,23 +119,24 @@ func TestMultitenant(t *testing.T) {
 					assert.Equal(fmt.Sprintf("%s.svc.id.goog", clusterProjectID), membershipOp.Get("authority.workloadIdentityPool").String(), fmt.Sprintf("Membership %s workloadIdentityPool should be %s.svc.id.goog", id, clusterProjectID))
 				}
 
-				// Bank of Anthos SA
-				saName := "bank-of-anthos"
-				workloadIdentitiesSAEmail := fmt.Sprintf("%s@%s.iam.gserviceaccount.com", saName, clusterProjectID)
-				saOp := gcloud.Run(t, fmt.Sprintf("iam service-accounts describe %s --project %s", workloadIdentitiesSAEmail, clusterProjectID))
-				assert.False(saOp.Get("disabled").Bool(), "Service account should not be disabled.")
-
-				workloadIdentitiesUsers := []string{
-					fmt.Sprintf("serviceAccount:%s.svc.id.goog[accounts-%s/bank-of-anthos]", clusterProjectID, envName),
-					fmt.Sprintf("serviceAccount:%s.svc.id.goog[ledger-%s/bank-of-anthos]", clusterProjectID, envName),
-				}
-
-				workloadIdentitiesSAIamFilter := "bindings.role:'roles/iam.workloadIdentityUser'"
-				workloadIdentitiesSAIamCommonArgs := gcloud.WithCommonArgs([]string{"--flatten", "bindings", "--filter", workloadIdentitiesSAIamFilter, "--format", "json"})
-				workloadIdentitiesSAPolicyOp := gcloud.Run(t, fmt.Sprintf("iam service-accounts get-iam-policy %s", workloadIdentitiesSAEmail), workloadIdentitiesSAIamCommonArgs).Array()[0]
-				workloadIdentitiesSaListMembers := utils.GetResultStrSlice(workloadIdentitiesSAPolicyOp.Get("bindings.members").Array())
-				assert.Subset(workloadIdentitiesSaListMembers, workloadIdentitiesUsers, fmt.Sprintf("service account %s should have workload identity users", workloadIdentitiesSAEmail))
-				assert.Equal(len(workloadIdentitiesUsers), len(workloadIdentitiesSaListMembers), fmt.Sprintf("service account % should have %d workload identity users", workloadIdentitiesSAEmail, len(workloadIdentitiesUsers)))
+				// App Service Accounts
+				for _, appName := range testutils.AppNames {
+					workloadIdentitiesSAEmail := fmt.Sprintf("%s@%s.iam.gserviceaccount.com", appName, clusterProjectID)
+					saOp := gcloud.Run(t, fmt.Sprintf("iam service-accounts describe %s --project %s", workloadIdentitiesSAEmail, clusterProjectID))
+					assert.False(saOp.Get("disabled").Bool(), "Service account should not be disabled.")
+
+					workloadIdentitiesUsers := []string{
+						fmt.Sprintf("serviceAccount:%s.svc.id.goog[accounts-%s/%s]", clusterProjectID, envName, appName),
```

Comment:
What is the point of using `appName` as the KSA name?  Would `default` be a better choice?

Reply:
This can be addressed later.

Reply:
This was a conversion of the existing hard coded `bank-of-anthos` service account, however I'm not certain it's actually used/required: @yliaog?

My preference would be to drop from this stage entirely.